src/services/api.js

```diff
@@ -0,0 +1,38 @@
+const BASE_URL =
+  "https://crudcrud.com/api/e723ab3502d74db18d02a98c5c5dbea1/todos";
```

Comment:
BASE_URL should be stored in a .env file. /todos is not part of the base URL it is part of the path.